### PR TITLE
OBSDOCS-982: COO 0.2.0 release notes

### PR DESCRIPTION
**Pull request by bburt-rh**
Version(s): 4.12+

Issue: https://issues.redhat.com/browse/OBSDOCS-982

Link to docs preview: https://74810--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/monitoring/cluster_observability_operator/cluster-observability-operator-release-notes.html#cluster-observability-operator-release-notes-0-2-0

QE review:
- [x] QE has approved this change.

Additional information:
This PR adds release notes for the Cluster Observability 0.2.0 release.

Please note that I can't add the direct link to the advisory until the release is "shipped live", so that will be a TBD for release day. In the meantime, if you could please review the current text, that'd be great.

### CODE REVIEW

File: observability/monitoring/cluster_observability_operator/cluster-observability-operator-release-notes.adoc [hunk 1]
```diff
@@ -16,6 +16,16 @@ The {coo-short} complements the built-in monitoring capabilities of {product-tit
...
 
+[id="cluster-observability-operator-release-notes-0-2-0"]
+== {coo-full} 0.2.0
+The following advisory is available for {coo-full} 0.2.0:
+
+* RHEA-2024:nnnn Cluster Observability Operator 0.2.0 - LINK TO BE ADDED WHEN THE ADVISORY IS LIVE
+
+[id="cluster-observability-operator-0-2-0-new-features-enhancements"]
+=== New features and enhancements
+* With this release, the {coo-full} supports installing and managing observability-related plugins for the {product-title} web console user interface (UI). (link:https://issues.redhat.com/browse/COO-58[*COO-58*])
```
Comment (lihongyan1) [inline observability/monitoring/cluster_observability_operator/cluster-observability-operator-release-notes.adoc:27]: change 'observability-related plugins' to 'console-dashboards-plugin' for this release only support  'console-dashboards-plugin'

Comment (danielmellado) [reply observability/monitoring/cluster_observability_operator/cluster-observability-operator-release-notes.adoc:27]: I agree with that and keeping it generic tbh, for the very same reasons. Let's see Gabriel's thoughts.

Comment (jgbernalp) [reply observability/monitoring/cluster_observability_operator/cluster-observability-operator-release-notes.adoc:27]: Yes this is generic, as the UIPlugin is an API for observability plugins not only specific to the dashboards.

**Pull request merged**